svensk-exegetisk-arsbok.csl

```diff
@@ -0,0 +1,782 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="sv-SE">
```

Comment:
given that they accept manuscripts in "Swedish, Danish, Norwegian, English, German and French." wouldn't it make more sense to remove the default locale?